action.yml

```diff
@@ -55,6 +55,10 @@ inputs:
     description: Branch for finalizing releases
     required: false
     default: ""
+  multi_dockerignore:
+    description: Use service's dockerignore file
```

Comment:
@karaxuna after enabling external contributors, I now see a different error in CI
See: https://github.com/balena-io/deploy-to-balena-action/actions/runs/4505762867/jobs/7932156158#step:4:95

Can you try removing this `'` and see whether that fixes the yml?